retro/tables/pexp_5d.py

```diff
@@ -652,7 +698,7 @@ def get_llh_all_doms(
         num_operational_doms = len(event_dom_info)
         num_hits = len(event_hit_info)
 
-        recip_time_window = 1 / time_window
+        recip_time_window = 1. / time_window
```

Comment:
no need to do this since Python 2.6... but don't worry about reverting, just be aware of it in future.

Reply:
I know it's no necessary....you can accept these things as coding style....everybody has different preferences.....some people will even write 1.0, none of them are more or less correct, they are equivalent